docs/styles/styles.css

```diff
@@ -325,4 +325,30 @@
     the component is trying to set internally on #description */
     font-size: var(--rh-font-size-body-text-sm, 0.875rem) !important;
   }
+
+  /* Used in Foundations > Typography.
+  Removes margin below <p>s in table */
+  td.altered-text p,
+  td.not-altered-text p {
+    margin-block-start: 0;
+    margin-block-end: 0;
+  }
+
+  /* Used in Foundations > Typography.
+  Adds strikethrough and gray font color to first line in cell */
+  td.altered-text p:first-child {
+    color: var(--rh-color-gray-40, #a3a3a3);
+  }
+
+  /* Used in Foundations > Typography.
+  Changes text color to red */
+  td.altered-text p:last-child {
+    color: var(--rh-color-text-brand-on-light, #ee0000);
+  }
+
+  /* Used in Foundations > Typography.
+  Changes text color to green */
+  td.not-altered-text p:last-child {
+    color: var(--rh-color-green-60, #3d7317);
+  }
```

Comment:
Because this is specific to `foundations/typography/scale.md` I would add it as a page-specific style and possibly a special table style.

If this style becomes pervasive throughout the site then we'd move it to the patterns @layer in styles.css

Example:
```
<style>
rh-table td.altered-text p,
rh-table td.not-altered-text p {
  margin-block-start: 0;
  margin-block-end: 0;
}
```